### DISCUSSION TITLE
好像不能用8080之外的端口

### DISCUSSION THREAD

**Discussion by 955s** [messense/aliyundrive-webdav#833] [Q&A]
### 问题描述

root@SG2:~# docker run -d --name=aliyundrive-webdav --restart=unless-stopped -p 56315:56315 \
>   -v /etc/aliyundrive-webdav/:/etc/aliyundrive-webdav/ \
>   -e REFRESH_TOKEN='xx' \
>   -e WEBDAV_AUTH_USER=admin \
>   -e WEBDAV_AUTH_PASSWORD=aliyun \
>   messense/aliyundrive-webdav
75fe0226174ebcf8fa68608f00064b8573fea984c06a765a48c8207897703c59
root@SG2:~# netstat -tln
Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State
tcp        0      0 0.0.0.0:56315           0.0.0.0:*               LISTEN
tcp        0      0 0.0.0.0:2969            0.0.0.0:*               LISTEN
tcp        0      0 127.0.0.1:45483         0.0.0.0:*               LISTEN
tcp        0      0 0.0.0.0:80              0.0.0.0:*               LISTEN
tcp        0      0 127.0.0.53:53           0.0.0.0:*               LISTEN
tcp6       0      0 :::56315                :::*                    LISTEN
tcp6       0      0 :::2969                 :::*                    LISTEN
tcp6       0      0 :::80                   :::*                    LISTEN
tcp6       0      0 :::25069                :::*                    LISTEN
tcp6       0      0 :::443                  :::*                    LISTEN
root@SG2:~# wget 127.0.0.1:56315
--2023-05-26 11:06:38--  http://127.0.0.1:56315/
Connecting to 127.0.0.1:56315... connected.
HTTP request sent, awaiting response... No data received.
Retrying.

--2023-05-26 11:06:39--  (try: 2)  http://127.0.0.1:56315/
Connecting to 127.0.0.1:56315... connected.
HTTP request sent, awaiting response... No data received.
Retrying.

^C
root@SG2:~#

### 重现步骤

好像不能用8080之外的端口

### 版本

docke安装

### 运行平台

Linux

### 日志

root@SG2:~# docker run -d --name=aliyundrive-webdav --restart=unless-stopped -p 56315:56315 \
>   -v /etc/aliyundrive-webdav/:/etc/aliyundrive-webdav/ \
>   -e REFRESH_TOKEN='xx' \
>   -e WEBDAV_AUTH_USER=admin \
>   -e WEBDAV_AUTH_PASSWORD=aliyun \
>   messense/aliyundrive-webdav
75fe0226174ebcf8fa68608f00064b8573fea984c06a765a48c8207897703c59
root@SG2:~# netstat -tln
Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State
tcp        0      0 0.0.0.0:56315           0.0.0.0:*               LISTEN
tcp        0      0 0.0.0.0:2969            0.0.0.0:*               LISTEN
tcp        0      0 127.0.0.1:45483         0.0.0.0:*               LISTEN
tcp        0      0 0.0.0.0:80              0.0.0.0:*               LISTEN
tcp        0      0 127.0.0.53:53           0.0.0.0:*               LISTEN
tcp6       0      0 :::56315                :::*                    LISTEN
tcp6       0      0 :::2969                 :::*                    LISTEN
tcp6       0      0 :::80                   :::*                    LISTEN
tcp6       0      0 :::25069                :::*                    LISTEN
tcp6       0      0 :::443                  :::*                    LISTEN
root@SG2:~# wget 127.0.0.1:56315
--2023-05-26 11:06:38--  http://127.0.0.1:56315/
Connecting to 127.0.0.1:56315... connected.
HTTP request sent, awaiting response... No data received.
Retrying.

--2023-05-26 11:06:39--  (try: 2)  http://127.0.0.1:56315/
Connecting to 127.0.0.1:56315... connected.
HTTP request sent, awaiting response... No data received.
Retrying.

^C
root@SG2:~#

**Comment by newcardriver** [comment]
我刚用compose的方式搭建了一个监听在非8080端口的，目测还是可以。

**Comment by ischaojie** [comment]
-p 56315:8080
